plugin.json

```diff
@@ -21,50 +21,50 @@
       {
         "key": "DisplayMode",
         "type": "radio",
-        "display_name": "Display the GIF as an",
+        "display_name": "Display the GIF as:",
         "default": "embedded",
         "options": [
           {
             "display_name": "Embedded image (the GIF cannot be collapsed)",
             "value": "embedded"
           },
           {
-            "display_name": "Collapsable image preview (the full URL is displayed, requires links preview to be activated)",
+            "display_name": "Collapsible image preview (the full URL is displayed, requires link previews to be enabled)",
             "value": "full_url"
           }
         ],
-        "help_text": "It is not yet possible to collapse an embedded image in Mattermost: use the Full URL option if preferred and keep an eye on [this issue](https://github.com/moussetc/mattermost-plugin-giphy/issues/12).\n\n**To activate links preview**, go to System Console > Posts > Enable Link Previews."
+        "help_text": "It is not yet possible to collapse an embedded image in Mattermost: use the Full URL option if preferred and keep an eye on [this issue](https://github.com/moussetc/mattermost-plugin-giphy/issues/12).\n\n To enable link previews, go to **System Console > Site Configuration > Posts > Enable Link Previews**."
       },
       {
         "key": "Provider",
         "type": "radio",
-        "display_name": "GIF Provider",
+        "display_name": "GIF Provider:",
```

Comment:
```suggestion
        "display_name": "GIF Provider",
```

Reply:
Woops _this line_ does need a colon, I put my comment at the wrong line, sorry.